src/lib/auth/user-resolver.ts

```
@@ -0,0 +1,172 @@
+  switch (resource) {
+    case 'conversations':
+      current = await prisma.conversation.count({
+        where: {
+          agencyId: tenantContext.agencyId,
+          deletedAt: null
+        }
+      })
+      break
+    case 'orders':
+      // For orders, check monthly limit
+      const startOfMonth = new Date()
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)